packages/react-devtools-shared/src/devtools/views/Components/TreeContext.js

```diff
@@ -914,7 +919,9 @@ function TreeContextController({
 
     // Inspection element panel
     inspectedElementID:
-      defaultInspectedElementID == null ? null : defaultInspectedElementID,
+      defaultInspectedElementID != null
```

Comment:
It's a bit weird. why isn't it:
```
store.lastSelectedHostInstanceElementId ? store.lastSelectedHostInstanceElementId : defaultInspectedElementID == null ? null : defaultInspectedElementID?
```

Reply:
https://github.com/facebook/react/blob/ec98d36c3a47e9f02c45cc14ff6046fff8c27458/packages/react-devtools-shared/src/devtools/views/Components/TreeContext.js#L824-L828

Reply:
Could we rename it to `forceSelected...`?